src/piracy.jl

```diff
@@ -0,0 +1,119 @@
+module Piracy
+
+using Test: @test
+
+const Callable = Union{Function,Type}
```

Comment:
Won't this cause issues for functors?
I think this is just used for container type. And that won't be any faster than `Vector{Any}`, will it?

Reply:
You are right. I assumed this would be faster, but it appears not to be. Perhaps by far most objects in modules are functions anyway. I'll change it to Any.